docs/methods/authorizeCoinJoin.md

```diff
@@ -0,0 +1,61 @@
+## Bitcoin: authorize CoinJoin
+Allow device to do preauthorized operations in `signTransaction`, `getOwnershipProof` and `getOwnershipId` methods without further user interaction.
+
+Permission persists until device disconnection or `maxTotalFee` limit is reached.
+
+ES6
+```javascript
+const result = await TrezorConnect.authorizeCoinJoin(params);
+```
+
+CommonJS
+```javascript
+TrezorConnect.authorizeCoinJoin(params).then(function(result) {
+
+});
+```
+
+> :warning: **This feature is experimental! Do not use it in production!**
+
+> :warning: **Supported only by Trezor T with Firmware 2.4.4 or higher!**
+
+### Params
+[****Optional common params****](commonParams.md)
+#### Exporting single id
+* `path` — *required* `string | Array<number>` minimum length is `3`. [read more](path.md)
+* `maxTotalFee` — *required* `number`
+* `feePerAnonymity` — *optional* `number`
+* `coin` — *optional* `string` determines network definition specified in [coins.json](../../src/data/coins.json) file. Coin `shortcut`, `name` or `label` can be used. If `coin` is not set API will try to get network definition from `path`.
+* `scriptType` — *optional* `InputScriptType`
+* `amountUnit` — *optional* `AmountUnit`
+
+
+### Example
+Display id of the first bitcoin address:
+```javascript
+TrezorConnect.authorizeCoinJoin({
+    path: "m/86'/'/0'/0/0",
```

Comment:
1. `m/86'/'/0'/0/0` -> `m/86'/0'/0'/0/0`, this typo also appears in many places below.
2. The `scriptType` should be specified as Taproot, given that the purpose is 86, otherwise Trezor will show a warning.